docs/usage-with-rancher.md

```diff
@@ -1,20 +1,15 @@
 ## Usage With Rancher
 
-Currently, `docker-machine-opentelekomcloud` is not a part of the Rancher,
-so additional steps required to use it as a node driver
-
+Currently, `docker-machine-opentelekomcloud` is not a part of the Rancher, so additional steps required to use it as a node driver
 
 ### Remove old node driver:
 
- * Open Rancher UI page and go to `Tools` → `Drivers` → `Node Drivers`
+ * Open Rancher UI page and go to `Tools` -> `Drivers` -> `Node Drivers`
```

Comment:
Why replace `→` with `->`?

Reply:
I thought that it would be better insted of symbol

Reply:
Originally it was used to look better than `->` 🙂

Reply:
done